plugins/Colorizer/ColorizerFrontendControl.js

```diff
@@ -0,0 +1,109 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @flow
+ */
+
+'use strict';
+
+const React = require('react');
+const immutable = require('immutable');
+
+var decorate = require('../../frontend/decorate');
+
+import type {
+  ControlState,
+} from './ColorizerTypes';
+
+type Props = {
+  state: any,
+  onChange: (v: ControlState) => void,
+};
+
+type State = StateRecord;
+
+type DefaultProps = {};
+
+const StateRecord = immutable.Record({
+  enabled: false,
+});
+
+class ColorizerFrontendControl extends React.Component {
```

Comment:
Can you create and use an abstract component to avoid the duplication between `BananaSlugFrontendControl` and `ColorizerFrontendControl`?